strawberryfields/ops.py

```diff
@@ -1052,6 +1053,23 @@ def _apply(self, reg, backend, shots=1, **kwargs):
         return backend.measure_fock(reg, shots=shots, select=self.select, **kwargs)
 
 
+class MeasureThreshold(Measurement):
+    """:ref:`photo_detection`: measures a set of modes in the thresholded Fock basis, i.e.
+    zero or nonzero photons.
+
+    After measurement, the modes are reset to the vacuum state.
+    """
+    ns = None
+
+    def __init__(self, select=None):
+        if select is not None and not isinstance(select, Sequence):
+            select = [select]
+        super().__init__([], select)
```

Comment:
In this case, would `select` be a list of integers, i.e., `[0, 0, 1, 1, 0, 1]`?

Reply:
I guess it would. This would be useful once `MeasureThreshold` is implemented in the Fock backends. In the gaussian backend if you try to postselect you will get an error message.